x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result.scss

```
@@ -0,0 +1,65 @@
+.appSearchResultContainer {
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  width: 100%;
+}
+
+.appSearchResult {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__contentWrap {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    padding: $euiSize;
+    display: grid;
+    grid-template-rows: 1fr $euiSize * 1.25 /* 20px */;
+    border-right: 1px solid $euiColorLightShade;
```

```suggestion
    border-right: $euiBorderWidthThin solid $euiColorLightShade;
```

---

Oh oops actually [EUI has a whole combined var for this](https://github.com/elastic/eui/blob/master/src/global_styling/variables/_borders.scss)!

```suggestion
    border-right: $euiBorderThin;
```